server/database/index.ts

```diff
@@ -350,13 +376,21 @@ export const addUser = async (user: userObj) => {
           age: age
         }
       );
-
+      return newUser;
   }
   catch (err) {
     console.error('already added');
   }
 };
 
+export const updateUser = async (updateElement: any, userId?: number) => {
+  try {
+    const updatedUser = await User.update({ where: { id: userId }})
```

Comment:
https://sequelize.org/main/manual/model-querying-basics.html#simple-update-queries
first argument of `update()` is the info you want to update, second argument is the options (like where)

Reply:
Updated to use the `.update(updateElement, { where: { id: userId }})` that we talked about.